cmd/tracee-rules/output_test.go

```diff
@@ -112,26 +113,30 @@ HostName: foobar.local
 	}
 
 	for _, tc := range testCases {
-		var actualOutput bytes.Buffer
-		findingCh, err := setupOutput(&actualOutput, "", "", "", tc.outputFormat)
-		require.NoError(t, err, tc.name)
+		t.Run(tc.name, func(t *testing.T) {
+			actualOutput := NewSyncBuffer([]byte{})
```

Comment:
That was the root cause of DATA RACE warning it test code. bytes.Buffer is not thread safe, whereas it was access by two goroutines.